### DISCUSSION TITLE
Any experience with TF-CCTV?

### DISCUSSION THREAD

**Discussion by moskovskiy82** [blakeblackshear/frigate#9079] [General]
Found a thread which is almost 2 years old. 
A drop in instruction worked.
https://codeberg.org/Curid/TF-CCTV/issues/72
Running it only for one day and wonder if anybody can share their experience over default frigate model

**Comment by megakid** [comment]
Did you get good results? Interested to hear how the model is evolving

**Reply by moskovskiy82** [reply]
Not really. The detection was close to the stock one. And got a little bit more false positives. Looking at the repo - project seems to be abandoned. Pity.
